src/node_buffer.cc

```diff
@@ -1094,6 +1130,8 @@ void SetupBufferJS(const FunctionCallbackInfo<Value>& args) {
   env->SetMethod(proto, "hexWrite", HexWrite);
   env->SetMethod(proto, "ucs2Write", Ucs2Write);
   env->SetMethod(proto, "utf8Write", Utf8Write);
+  env->SetMethod(proto, "swap16", SwapShort);
+  env->SetMethod(proto, "swap32", SwapLong);
```

Comment:
While case differs, generally the names are the same between c++ and js. Would `Swap{16,32}` work?